src/lang/std/sketchcombos.ts

```diff
@@ -86,9 +87,7 @@ function createCallWrapper(
   }
 
   return {
-    // @ts-ignore
-    // Typescript can't tell args cannot have an Error in it.
-    callExp: createCallExpression(a, args),
+    callExp: createCallExpression(a, argsWOutErr),
```

Comment:
just voicing my opinion, I've already merged all this code at the time of comment: I'm not a fan of the type gymnastics you had to do to satisfy this but that's all I'll say - transforming data like this at runtime to satisfy a compile time check because the compiler is not smart enough I think should instead warrant an `as`